lib/web-llm.tsx

```diff
@@ -0,0 +1,99 @@
+import {
```

Comment:
Would love to add this directly to LangChain.js - will open a PR to this repo first with streaming support and see how that goes.